GPflowOpt/models.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2017 Joachim van der Herten
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from GPflow.param import Parameterized
+from GPflow.model import Model
+
+
+class ModelWrapper(Parameterized):
+    """
+    Class for fast implementation of a wrapper for models defined in GPflow. Once wrapped, all lookups for attributes
```

Comment:
lets make the first paragraph a one liner (so it is listed nicely if it ever becomes in the docs). So "Once..." is a new paragraph